.github/workflows/mkdocs.yml

```diff
@@ -0,0 +1,26 @@
+name: publish-docs
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'mkdocs.yml'
+      - 'docs/**'
+
+permissions:
+  contents: write
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
These should be pinned to hashes for consistency. Implemented here -https://github.com/RedHatProductSecurity/trestle-bot/pull/242. Realizing that this should be added to the contributing guide with suggested tools.

Reply:
Issue added here to make sure the docs are updated https://github.com/RedHatProductSecurity/trestle-bot/issues/317